contents/docs/getting-started/install.mdx

```diff
@@ -17,6 +17,6 @@ import Install from "./\_snippets/install.mdx"
 
 #### Set up a reverse proxy (optional)
 
-We recommend setting up a reverse proxy so that events are less likely to be intercepted by tracking blockers. We have our [own managed reverse proxy service included in the Teams plan](/docs/advanced/proxy/managed-reverse-proxy), which routes through our infrastructure and makes setting up your proxy easy. 
+We recommend setting up a reverse proxy so that events are less likely to be intercepted by tracking blockers. We have our [own managed reverse proxy service available for free](/docs/advanced/proxy/managed-reverse-proxy), which routes through our infrastructure and makes setting up your proxy easy. 
```

Comment:
```suggestion
We recommend setting up a reverse proxy so that events are less likely to be intercepted by tracking blockers. 

We have our [own managed reverse proxy service available for free](/docs/advanced/proxy/managed-reverse-proxy), which routes through our infrastructure and makes setting up your proxy easy. 
```

Reply:
For scanability